### PR TITLE
A logic condition as described in Issue #2687 was showing subscribe when it shouldn't

### PR DESCRIPTION
**Pull request by meuhland**
Fixed the logic by reusing condition that was causing the unknown error.

Enjoy!

### CODE REVIEW

**Comment by meuhland** [comment]
That's what you get for using vi to fix code right on the server...

**Comment by meuhland** [comment]
I'll repost without the garbage from me hitting the wrong button

File: main/course_info/about.php [hunk 1]
```diff
@@ -124,7 +124,16 @@
 ];
 
 $subscriptionUser = CourseManager::is_user_subscribed_in_course($userId, $course->getCode());
-
+$course_table = Database::get_main_table(TABLE_MAIN_COURSE);
```
Comment (jmontoyaa) [inline main/course_info/about.php]: You can use:

`$courseInfo = api_get_course_info_by_id($courseId);`

Then later you can check the subscribe value:

`$courseInfo['subscribe']`

Comment (meuhland) [reply main/course_info/about.php]: Awesome then it's even simpler to fix if this return a boolean.

I'd just be a assign and the same change in the template.

**Pull request closed without merging**